README.md

```diff
@@ -61,3 +61,24 @@ scripts are more niche but a simple "hello world" example looks like this:
     realityEditor.addons.addCallback('init', initService);
 }());
 ```
+
+## Using the Kepware Interface
+
+The Kepware Interface is compatible with both [KEPServerEX](https://www.kepware.com/en-us/products/kepserverex/) and [Kepware Edge](https://www.kepware.com/en-us/products/thingworx-kepware-edge/).
+
+### Kepware Connection Instructions
+1. Enter the Hardware Interfaces tab on the Vuforia Spatial Edge Server Web UI.
+![](interfaces/kepware/images/kepA.png)
+2. Enable the Kepware interface.
+![](interfaces/kepware/images/kepB.png)
+3. Enter in your connection details and save your changes.
+![](interfaces/kepware/images/kepC.png)
+4. On your KEPServerEX or Kepware Edge server, browse through the rejected certificate list and accept the most recent certificate. This process differs between the two servers, so consult the corresponding User Manual.
```

Comment:
If possible linking to the user manuals or a place to get the user manual would improve this

Reply:
Unfortunately it seems like it's impossible to get a long-lasting static link to either user manual. I bookmarked the Edge manual when setting it up but the link became invalid within a day. Do you think it would be useful to link to https://www.kepware.com/en-us/products/kepserverex/ and https://www.kepware.com/en-us/products/thingworx-kepware-edge/ and tell people to scroll down to find the user manual themselves?

Reply:
Yeah I'd say anything is better than nothing. Less than ideal that it can't be directly linked